### ISSUE TITLE
오픈소스 라이브러리들을 SettingView에 나타내어야 합니다.

### ISSUE THREAD

**Issue by enghqii** [GDG-SSU/deviewsched-android#11]
Deview는 그 자체로도 오픈소스프로젝트이지만, 다양한 오픈소스를 활용하고있습니다. 해당 오픈소스 라이브러리들을 SettingView에 나타내어야 합니다.

다른 어플리케이션들을 참고해서 `ui.setting.OSSlicensesFragment` 를 꾸미면 될것 같네요

**Comment by ghost** [comment]
예, 대부분의 오픈소스 라이브러리를 maven으로 사용하고있기에 오픈소스 라이브러리의 사용이 잘 명시되어있고, `util.LogUtils`와 같이 클래스 소스를 직접 가져와서 쓰는경우에는 따로 기록을 해두면서 사용하고있습니다. 프로젝트 말미즈음에 챙기면 될것같네요.
